### DISCUSSION TITLE
"Add Media" NOT responding for a PODS Custom Setting with a WYSIWYG field set with oEmbed, Shortcodes and Add Media enabled

### DISCUSSION THREAD

**Discussion by papa-pup** [pods-framework/pods#6663] [Q&A] [answered]
### Description

"Add Media" for a PODS Custom Setting with a WYSIWYG field set with oEmbed, Shortcodes and Add Media enabled.

However, with the same PODS Custom Setting with added another field - i.e. Image, Add Media works.

Setting the added another field with a Hidden UI, Add Media will not work

### Version

2.8.17

### Testing Instructions

1. Create a PODS Custom Setting with a WYSIWYG field set with oEmbed, Shortcode and Add Media enabled.

### Possible Workaround

Downgrade to PODS v2.7.31 - issue resolved.

**Comment by papa-pup** [comment]
@sc0ttkclark / @JoryHogeveen , can you reconfirm if this is indeed a bug? Is this already a know issue?

**Comment by JoryHogeveen** [comment]
Hi @papa-pup 
It looks like this is related to #6283 , can you confirm?

**Comment by papa-pup** [comment]
Hi @JoryHogeveen 
This is more of the "Add Media" did not respond at all with a single WYSIWYG field or single WYSIWYG field with another field.

I have **NOT** met with the issue as describe on #6283.

By the way, I tested again on 2.8.19 to 20 and the problem still persist.

**Comment by papa-pup** [comment]
@JoryHogeveen , any update to any fix will be done?

**Comment by JoryHogeveen** [comment]
Hi @papa-pup 

I tested this again and still cannot reproduce this. No matter what I do, the "Add media" button opens the modal for images.
Please check for plugin and/or theme conflicts: https://docs.pods.io/faqs/plugin-theme-conflicts/

Cheers, Jory

**Comment by papa-pup** [comment]
[2022-07-05T15-49-13D2-47.webm](https://user-images.githubusercontent.com/59635968/177368565-c9c648c5-1f36-4a2a-9592-f1d7996ae363.webm)
@JoryHogeveen ,
I did a fresh install of WP with PODS (v2.8.23). I recorded this to show you exactly what I did and the error is reproduced.

 Please note - This is create a Custom SETTING and not a CPT.

**Comment by JoryHogeveen** [comment]
Hi @papa-pup 
Sorry for this! I totally missed that part. Issue reproduced!

**Comment by papa-pup** [comment]
@JoryHogeveen Brother,
No worries. We are here to make PODS better! 😄

**Comment by HemboPagi** [comment]
any news about that? is there any workaround (except using code view)?

**Reply by sc0ttkclark** [reply]
This sounds like it should have been a bug and not converted to a discussion when we did the batch convert this summer. Will do that and get it assigned to 2.9.5

**Comment by sc0ttkclark** [comment, marked as answer]
As far as I can tell, this is fixed in 2.9.x, or at least in 2.9.4 which I've been testing with. This should be all good after 2.9.4 is released.

**Reply by HemboPagi** [reply]
super, many thanks
